dbms/CMakeLists.txt

```diff
@@ -262,8 +263,10 @@ if (NOT USE_INTERNAL_ZSTD_LIBRARY)
 endif ()
 
 if (USE_QPL)
-    target_link_libraries (dbms ${QPL_LIBRARY})
-    target_include_directories (dbms PRIVATE ${QPL_INCLUDE_DIR})
+    target_include_directories (dbms BEFORE PRIVATE ${QPL_INCLUDE_DIR})
+    set_source_files_properties(
+        src/IO/Compression/CompressionCodecDeflateQpl.cpp
+        PROPERTIES COMPILE_FLAGS "-mwaitpkg")
```

Comment:
What does this compile_flags mean? Why is this required now but not in the previous build?

Reply:
```
_tpause(1, __rdtsc() + 1000);
```

https://www.felixcloutier.com/x86/tpause

Reply:
https://github.com/pingcap/tiflash/blob/5c2d39d6fb2c7eb55d28f2d0495a1a26f5030d23/dbms/src/IO/Compression/CompressionCodecDeflateQpl.cpp#L239

here